tests/factories.py

```diff
@@ -71,7 +71,6 @@ class Meta:
 
 
 class RoleFactory(factory.alchemy.SQLAlchemyModelFactory):
-    name = factory.Faker("text")
```

Comment:
@ammaratef45 can we add a Faker ``RoleType`` in case it's not provided?

Reply:
@amihaiemil I don't know how to make this or why it should be done.
Any hints?

Reply:
@ammaratef45 well, I thought the previous line (``name = factory.Faker("text")``) was exactly that: it provided a fake name in case it wasn't provided in the constructor.

If it's not that easy, then we can leave it like this :)